app/build.gradle.kts

```diff
@@ -0,0 +1,148 @@
+/*
+ * Copyright © 2014-2020 The Android Password Store Authors. All Rights Reserved.
+ * SPDX-License-Identifier: GPL-3.0-only
+ */
+import java.util.Properties
+
+plugins {
+    kotlin("android")
+}
+
+val keystorePropertiesFile = rootProject.file("keystore.properties")
+
+fun isSnapshot(): Boolean {
+    return System.getenv("GITHUB_WORKFLOW") != null && System.getenv("SNAPSHOT") != null
+}
+
+android {
+    /*
+    if (isSnapshot()) {
+        android.applicationVariants.all { variant ->
+            variant.outputs.all {
+                outputFileName = "aps-${variant.getFlavorName()}_${defaultConfig.versionName}.apk"
+            }
+        }
+    }
+    */
```

Comment:
@FabianHenneke do you have an idea on how to do this in the Kotlin DSL?

Reply:
I have personally never used the Kotlin DSL, but this may be relevant: https://stackoverflow.com/questions/50792428/how-to-access-variant-outputfilename-in-kotlin